.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       shell: bash
     - name: Run tests on Ubuntu/Windows
       if: matrix.os != 'macOS-latest'
-      run: cargo test --workspace
+      run: cargo test --workspace --features unstable-test
```

Comment:
in addition to this I think we should also run: `RUSTFLAGS='--deny warnings' cargo check --features unstable-test` (there's a previous build step that does this without the --features)